Composer/packages/client/src/components/BotConvertDialog.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import { OpenConfirmModal } from '@bfc/ui-shared';
+import formatMessage from 'format-message';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+
+const contentContainer = css`
+  max-width: 444px;
+`;
+
+export const BotConvertConfirmDialog = () => {
+  return OpenConfirmModal(formatMessage('Convert your project to the latest format'), '', {
+    confirmText: formatMessage('Convert'),
+    onRenderContent: () => (
+      <div css={contentContainer}>
+        <p>
+          {formatMessage(
+            'This project was created in an older version of Composer. To open this project in Composer 2.0, we must copy your project and convert it to the latest format. Your original project will not be changed.'
+          )}
+        </p>
+        <p>
+          {formatMessage('If you have created custom components, you might need to rebuild them. ')}
```

Comment:
Do we want to always show this, or only show this when customized actions are detected?

Reply:
use formatMessage.rich, currently you are missing localizing ```Learn more about custom components.```

Reply:
Ideally, only custom actions are detected. 
(As a fallback, if we can't detect if a bot has custom action, show if the message if the 1.4 bot has ejected runtime, and if we can't detect if the bot has ejected runtime, show this message all the time.)

Reply:
I updated this logic, only when the old bot has the ejected runtime will show this message. It's not easy to detect the custom actions before open the new bot.